vnext/Microsoft.ReactNative/Modules/NativeUIManager.cpp

```diff
@@ -30,8 +32,29 @@ using namespace facebook::react;
 
 namespace Microsoft::ReactNative {
 
-static YogaNodePtr make_yoga_node(YGConfigRef config) {
-  YogaNodePtr result(YGNodeNewWithConfig(config));
+static YGConfigRef make_yoga_config(winrt::Microsoft::ReactNative::ReactContext const &context, YGLogger logger) {
```

Comment:
nit: should this be camelcase for consistency?

Reply:
lol. It probably should be, but I was aiming for consistency with the existing `make_yoga_node` method, rather than the rest of react-native-windows.